README.md

```diff
@@ -129,12 +128,14 @@ See our contributions policy and list of contributors to FlamingPy [here](https:
 
 ## Support
```

Comment:
@soosub, I had to change this part to be consistent with other Xanadu project formatting. Thanks for your initial commits.